examples/spark/spark-gluster/spark-master-service.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
@copejon the "---" is not used in the other Kubernetes YAML examples. I'd recommend you remove it and re-validate it works and just move on.